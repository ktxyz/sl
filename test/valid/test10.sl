integer count;
string message;

count := 5;
message := "Hello World";

print(message);
print(count);

for count := 1 to 3 do
    print(concatenate("Iteration: ", message));

if count > 2 then begin
    print("Count is greater than 2");
end
else
    print("Count is small");