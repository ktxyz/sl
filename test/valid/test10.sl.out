Hello World
5
Iteration: Hello World
Iteration: Hello World
Iteration: Hello World
Count is greater than 2